modules/testJdoDiscriminator/pom.xml

```diff
@@ -0,0 +1,288 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Sorry for the lengthy presentation of testJdoDiscriminator in the gihub view.  It's not part of the production code but enables the instructions to work as written.